src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/NewEditCyclePage/PayoutsSection/hooks/usePayoutsTable.tsx

```diff
@@ -394,7 +394,7 @@ export const usePayoutsTable = () => {
 
   function handleDeleteAllPayoutSplits() {
     setDistributionLimit(0)
-    setPayoutSplits([])
+    _setPayoutSplits([])
```

Comment:
Sneaks in a very small bug when clicking "Delete all payouts" on payouts table still results in the form thinking there is no diff -> can't open "Review and confirm" modal